src/screens/Settings/element/ProfileSettingItem.js

```diff
@@ -28,7 +28,7 @@ const styles = StyleSheet.create({
   card: {
     height: 52,
     borderRadius: 8,
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the card has been changed from light grey to gray100. Ensure this color change aligns with the overall theme and doesn't affect readability or user experience.
```diff
- backgroundColor: COLORS.lightgrey,
+ backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->